### ISSUE TITLE
which is the version of opencv that you use?

### ISSUE THREAD

**Issue by zengletian1491** [KAIST-VCLAB/texturefusion#2]
Hello. Which is the version of opencv that you use? Which is the version of cmake that you use?
They should be matched with microsoft visual studio 2013. What's more, cmake of lower version
can not compile on 64-bit Windows system. All these factors make it difficult for us to compile this
project. Please update your codes to make it compile in a single project if you like.

**Comment by pavan4** [comment]
FWIW, I was able to compile against `Opencv 4.4.0` with `VS2019` on `Win 10` with some minor code edits.

Added a PR with the code edits and a VS project you can compile here : https://github.com/KAIST-VCLAB/texturefusion/pull/3

**Comment by Mronne** [comment]
@pavan4 
Can you successfully run its default data? I can compile but there are still many problems running

**Comment by pavan4** [comment]
Yes I was able to run. What problems are you seeing? What gpu are you running this on?

**Comment by Mronne** [comment]
> Yes I was able to run. What problems are you seeing? What gpu are you running this on?

I'm running on GTX 1060 3GB. It has illegal Memory Access on cudaDeviceSynchronize.

**Comment by johntranz** [comment]
Is there a detailed guide on compiling the project to run it (video or documentation), because I don't know where to start. Can the project run on AMD GPUs? I found the real-time demo video very fast.
